example/bwmf/bwmf_task.go

```diff
@@ -393,7 +407,7 @@ func (t *bwmfTask) GetDShard(ctx context.Context, request *pb.Request) (*pb.Resp
 	return resp, nil
 }
 
-func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, linkType, meta string) {
+func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, meta string) {
```

Comment:
Linktype should be app logic specific thing.

Take Parents/Children for example, in application, if A send data to "parent" B, B knows it's from "children" A.